command_line/find_bad_pixels.py

```diff
@@ -133,29 +94,84 @@ def run(args):
         else:
             total += peak_pixels.as_1d().as_int()
 
-    p.finished("Finished finding hot pixels on %d images" % len(images))
+    return total
+
+
+def run(args):
+    usage = "dev.dials.find_bad_pixels [options] data_master.h5"
+
+    parser = OptionParser(
+        usage=usage,
+        phil=phil_scope,
+        read_experiments=True,
+        read_experiments_from_images=True,
+        epilog=help_message,
+    )
+
+    params, options = parser.parse_args(show_diff_phil=True)
+
+    experiments = flatten_experiments(params.input.experiments)
+    if len(experiments) != 1:
+        parser.print_help()
+        print("Please pass an experiment list\n")
+        return
+
+    imagesets = experiments.imagesets()
+
+    if len(imagesets) != 1:
+        raise Sorry("Please pass an experiment list that contains one imageset")
+
+    imageset = imagesets[0]
+    detectors = imageset.get_detector()
```

Comment:
Doesn't this give a list of panels, not detectors? In which if it's a panel, call it a panel!